articles/purview/tutorial-using-python-sdk.md

```diff
@@ -0,0 +1,335 @@
+---
+title: "How to use Azure Purview Python SDK"
+description: This tutorial describes how to use the Azure Purview Python SDK to scan data and search the catalog.
+author: raoufaliouat
+ms.author: raoufaliouat
+ms.service: purview
+ms.subservice: purview-data-scanning
+ms.topic: tutorial
+ms.date: 01/10/2022
+
+# Customer intent: I can use the scanning and catalog Python SDKs to perform CRUD operations on data sources and scans, trigger scans and also to search the catalog.
+---
+
+
+
+# Tutorial: Use the Purview Python SDK
+
+The aim of this tutorial is to introduce you to the use of the Azure Purview Python SDK to programmatically do the most common operations you can find in Azure Purview Studio.
+
+In this tutorial, you will learn how to:
+
+*   Grant the required rights to work programmatically with Purview
+*   Register a Blob Storage container as a data source in Purview
+*   Define and run a scan
+*   Search the catalog
+*   Delete a data source
+
+## Prerequisites
+
+For this tutorial, you will need :
```

Comment:
For each of these, add a link the customer can follow if they DON'T have one of these prerequisites.

You can use the SQL Server tutorial as an example: https://docs.microsoft.com/en-us/azure/purview/tutorial-register-scan-on-premises-sql-server#prerequisites